Source/WebCore/platform/graphics/egl/GLContext.cpp

```diff
@@ -362,38 +368,53 @@ std::unique_ptr<GLContext> GLContext::create(GLNativeWindowType window, Platform
     }
 
     EGLContext eglSharingContext = platformDisplay.sharingGLContext() ? static_cast<GLContext*>(platformDisplay.sharingGLContext())->m_context : EGL_NO_CONTEXT;
-    if (platformDisplay.type() == PlatformDisplay::Type::Surfaceless) {
-        auto context = createSurfacelessContext(platformDisplay, eglSharingContext);
-        if (!context)
-            WTFLogAlways("Could not create EGL surfaceless context: %s.", lastErrorString());
-        return context;
+    auto context = createWindowContext(window, platformDisplay, eglSharingContext);
+    if (!context)
+        WTFLogAlways("Could not create EGL context.");
```

Comment:
What's the reason to remove the last error string here? I suppose this is because `createWindowContext()` already logs the error itself.